radicle-httpd/src/api/v1/projects.rs

```diff
@@ -722,17 +774,17 @@ mod routes {
             response.json().await,
             json!({
                 "entries": [
-                  {
-                    "path": "README",
-                    "name": "README",
-                    "lastCommit": null,
-                    "kind": "blob"
-                  },
                   {
                     "path": "dir1",
                     "name": "dir1",
                     "lastCommit": null,
                     "kind": "tree"
+                  },
+                  {
+                    "path": "README",
+                    "name": "README",
+                    "lastCommit": null,
+                    "kind": "blob"
```

Comment:
Now `repo.tree()` returns a tree with entries sorted.  The sort always lists the directories (trees) first.